### DISCUSSION TITLE
When adding tailwind classes, the components are not being changed

### DISCUSSION THREAD

**Discussion by Lucija360** [tailwindlabs/tailwindcss#7504] [Help]
**What version of Tailwind CSS are you using?**

3.0.22

**What build tool (or framework if it abstracts the build tool) are you using?**

 Next.js 12.0.10, Webpack 5.69.0

**What version of Node.js are you using?**

v 16.13.2

**What browser are you using?**

Microsoft Edge

**What operating system are you using?**

Windows 10

**Reproduction URL**

![Tailwind css](https://user-images.githubusercontent.com/89902545/154472545-853e6b3c-9b16-49a8-9291-50892ecd23ee.PNG)

**Describe your issue**

When adding tailwind classes, the components are not being changed like for example height and margin-bottom as seen in the picture uploaded up above

**Comment by adamwathan** [comment]
Hey! This sounds like a configuration issue, perhaps with your `content` configuration. Either way not possible to help without a proper reproduction (a project someone can clone and run and see the issue) — screenshot of a React component isn't enough I'm afraid, need the full picture.

**Reply by Lucija360** [reply]
https://github.com/Lucija360/Hulu.git

**Reply by NovusOpus** [reply]
@Lucija360 Yep, your `content` configuration is wrong. Currently you have: 
```JS
module.exports = {
  
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
}
```
But there's no `src` path in your project root, so to fix it you have to change your `tailwind.config.js` to this:

```JS
module.exports = {
  content: [
    './pages/**/*.{html,js,jsx}',
    './components/**/*.{html,js,jsx}',
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}
```

**Reply by Lucija360** [reply]
Yeah that fixed it, thank you!
